ams/server/src/main/java/com/netease/arctic/server/ArcticServiceContainer.java

```diff
@@ -183,37 +193,45 @@ private void initThriftService() throws TTransportException {
     String bindHost = serviceConfig.getString(ArcticManagementConf.SERVER_BIND_HOST);
 
     LOG.info("Starting thrift server on port:" + port);
+    TNonblockingServerSocket serverTransport = getServerSocket(bindHost, port);
 
-    TMultiplexedProcessor processor = new TMultiplexedProcessor();
-    final TProtocolFactory protocolFactory;
-    final TProtocolFactory inputProtoFactory;
-    protocolFactory = new TBinaryProtocol.Factory();
-    inputProtoFactory = new TBinaryProtocol.Factory(true, true, maxMessageSize, maxMessageSize);
-
-    ArcticTableMetastore.Processor<ArcticTableMetastore.Iface> tableMetastoreProcessor =
+    ArcticTableMetastore.Processor<ArcticTableMetastore.Iface> tableManagementProcessor =
         new ArcticTableMetastore.Processor<>(ThriftServiceProxy.createProxy(ArcticTableMetastore.Iface.class,
             new TableManagementService(tableService), ArcticRuntimeException::normalizeCompatibly));
-    processor.registerProcessor(Constants.THRIFT_TABLE_SERVICE_NAME, tableMetastoreProcessor);
+    tableManagementServer =
+        createThriftServer(tableManagementProcessor, Constants.THRIFT_TABLE_SERVICE_NAME, serverTransport,
+            maxMessageSize, Executors.newFixedThreadPool(workerThreads), selectorThreads, queueSizePerSelector);
```

Comment:
use `Executors.newFixedThreadPool(int nThreads, ThreadFactory threadFactory)` to set the thread name pattern for executors.